.github/workflows/build.yml

```diff
@@ -122,59 +124,29 @@ jobs:
           path: |
             store
             derivations
-          key: cache-store-${{ runner.os }}-${{github.run_id}}
+          key: cache-store-${{ runner.os }}-${{ github.run_id }}
 
-      - name: Prepare pdf files to be deployed to site
-        if: github.ref == 'refs/heads/master'
-        run: |
-          mkdir -p pdfs
-          for f in $(find -L outputs -name '*.pdf'); do
-            cp -L --force $f pdfs
-          done
-
-      - name: Prepare html docs to be deployed to site
-        if: github.ref =='refs/heads/master'
-        id: html
-        run: |
-          mkdir -p html
-          cp -L -r outputs/ledger/html html/ledger
-          cp -L -r outputs/midnight/html html/midnight
-          ledger=$(nix eval -f default.nix "ledger.name")
-          midnight=$(nix eval -f default.nix "midnight.name")
-          echo "ledger=$ledger" >> $GITHUB_OUTPUT
-          echo "midnight=$midnight" >> $GITHUB_OUTPUT
-
-      - name: Build static file
+      - name: Upload PDF artifacts
```

Comment:
Nice that we can remove all of this, but I don't get it - where are the pdfs created?

Reply:
Ah I think I see,  the `Build ledger` above  is already preparing the pdfs, right? with this :`rsync -r --exclude={'**/nix-support','**/lib'} outputs/ledger*/* docs/`.

Reply:
Everything gets done by the Makefile; the PDFs get crated by `make docs`. This is what you would do locally, but `nix` just wraps around the Makefile to provide a reproducible environment. So, these PDFs get created by `nix` in the end, which itself calls `make docs`.

Reply:
Cool, thanks for the explanation :+1: